### PR TITLE
V2

### PR DESCRIPTION
**Pull request by waciumawanjohi**

### CODE REVIEW

**Comment by waciumawanjohi** [comment]
Discussion of these changes should take place [in this issue](https://github.com/ossu/data-science/issues/61).

**Comment by waciumawanjohi** [comment]
Discussion of these changes should take place [in this issue](https://github.com/ossu/data-science/issues/61).

File: README.md [hunk 1]
```diff
@@ -16,288 +16,116 @@
...
+
+### Introduction to Computer Science
+Students who already know basic programming in any language can skip this course
+[Python for Everybody](https://www.py4e.com/) 
+[Introduction to Computer Science and Programming Using Python](https://www.edx.org/course/introduction-computer-science-mitx-6-00-1x-7)
+[Introduction to Computational Thinking and Data Science](https://www.edx.org/course/introduction-computational-thinking-data-mitx-6-00-2x-3)
+
+### Data Structures and Algorithms
+[Algorithms, Part I](https://www.coursera.org/learn/algorithms-part1)
+[Algorithms, Part II](https://www.coursera.org/learn/algorithms-part2)
```
Comment (mattjperez) [inline README.md:65]: These courses are completely in Java and steep learning curve from the start. Those coming straight from python/r/julia will have a hard time adjusting to both the course materials and programming syntax. Suggest an optional course on Java as pre-requisite, specifically (Java Programming Part 1 and 2)[https://java-programming.mooc.fi/] by University of Helsinki. Gives college credit for Finland residents.

Comment (waciumawanjohi) [reply README.md:65]: @mattjperez Cogent comment. Can you comment [in the issue](https://github.com/ossu/data-science/issues/61)?

**Pull request merged**